spec/rubocop/cop/rspec/multiple_subjects_spec.rb

```diff
@@ -19,6 +19,20 @@
         end
       end
     RUBY
+
+    expect_correction [
```

Comment:
any reason a heredoc couldn't be used here?

Reply:
Yeah, the code currently in line 27 (`      '  ',`) would contain trailing spaces if formatted as a heredoc.

Reply:
Guess we can improve the autocorrect by using `range_with_surrounding_space` but that can happen in another PR.
I wonder how it was working before with the `include_examples` though

Reply:
> I wonder how it was working before with the `include_examples` though

In the previous examples there are two `include_examples('autocorrect', …`. In the first example, two `subject`s are changed to `let`, and heredoc is used. In the second example, two unnamed `subject`s are *removed*, and the indentation is left behind as trailing spaces – and heredoc isn’t used in that example.

And yes, we might be able to improve it in another PR. But I would assume people will have cops guarding against trailing spaces, empty lines etc.